### ISSUE TITLE
Can we have a better pop-up UI as this not only shows the app name weirdly but also doesnt list all the similar apps (like browsers here) with clarity like LinkSheet app chooser?

### ISSUE THREAD

**Issue by ghost** [TrianguloY/UrlChecker#405]
### Describe your suggested feature

Can we have a better pop-up UI as this not only shows the app name weirdly but also doesnt list all the similar apps (like browsers here) with clarity like LinkSheet app chooser?

Like in browsers or video players (including different YT versions like revanced, newpipe, simpmusic), it doesnt show their specific names and app icons so for 2 different youtube, it shows youtube, so one is guessing which one it is. 

Also some color themes would be appreciated apart from simple light and dark. 

### Other details

![photo_2024-12-26_10-34-29](https://github.com/user-attachments/assets/0e5fb231-3d97-41e3-91ec-f76b78f530c3)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by salmanuc** [comment]
Suggestion on this topic
I think there should be an option in the module to show a row without names or column with names below the box for browsers whitelisted in the module setting while also allowing user to configure the order of browser. 

Something similar to this (velja for macos) for horizontal bar
![Image](https://github.com/user-attachments/assets/5d4bab04-aa62-42a0-9945-3ef6f238f01b)

 like this for column view (linksheet android)
![Image](https://github.com/user-attachments/assets/a394b846-1849-4da8-ae87-58c693fa72df)

**Comment by siddhpant** [comment]
I agree, the UI is atrocious if you want to use the app as an app chooser. I know the app was meant to be used as a sanitizer, but the chooser is already implemented which drew me to it and makes it a very powerful and novel app with least friction.
